lib/client/sse_client.dart

```diff
@@ -25,9 +25,13 @@ class SseClient extends StreamChannelMixin<String> {
 
   String _serverUrl;
 
+  Timer _errorTimer;
+
   /// [serverUrl] is the URL under which the server is listening for
   /// incoming bi-directional SSE connections.
-  SseClient(String serverUrl) {
+  SseClient(String serverUrl, {bool withRetry}) {
```

Comment:
synced offline - since all of our known use cases will want to use retry I think we can roll to version 3.0.0 and make it non-optional. This package shouldn't have an overwhelming amount of churn to bump.

Reply:
Done.